frontend/src/metabase/lib/urls/models.ts

```diff
@@ -0,0 +1,36 @@
+import slugg from "slugg";
+import type { Card } from "metabase-types/api";
+import type { Card as LegacyCard } from "metabase-types/types/Card";
+import { question, QuestionUrlBuilderParams } from "./questions";
+import { appendSlug } from "./utils";
+
+type CardOrSearchResult = (Partial<Card> | Partial<LegacyCard>) & {
+  id?: number | string;
+  card_id?: number | string;
+  name?: string;
+};
+
+export function model(
```

Comment:
Sorry, I'm not sure I'm following?

Reply:
Sorry. I mean the `model` function itself doesn't do anything different than the `question` at all, it only receives and passes along arguments in the exact same manner. So, I don't see the purpose of this function yet.

So I thought this could be a place where we could support linking to the model detail page, but is not yet implemented. However, on second thought, it could be for a better semantic use. Say it'd be better to call `model()` function for models rather than directly using `question()` even if the implementation is the exact same.